java/google/registry/rde/imports/RdeDomainImportAction.java

```diff
@@ -121,6 +123,13 @@ private RdeImportUtils getImportUtils() {
       return importUtils;
     }
 
+    private DnsQueue getDnsQueue() {
```

Comment:
Is there some advantage to having the helper method do lazy initialization, versus just initializing the field in the constructor and then directly referencing the field?  The latter seems a little simpler.